iocBoot/templates/st.cmd.template

```diff
@@ -0,0 +1,103 @@
+#!../../bin/rhel7-x86_64/{{binary_name}}
+
+< envPaths
```

Comment:
envPaths never gets built

Reply:
Not yet implemented, along with the other couple things in the original PR message.

Reply:
soryr, I've lost my ability to read

Reply:
This update is only broken into 4 PRs with many comments and a bunch of lines of code. Not sure it could get much more complicated... ;)